iotdb-cli/pom.xml

```diff
@@ -43,6 +43,11 @@
             <artifactId>iotdb-jdbc</artifactId>
             <version>0.8.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.iotdb</groupId>
+            <artifactId>IoTDB-cluster</artifactId>
+            <version>0.8.0-SNAPSHOT</version>
+        </dependency>
```

Comment:
i am afraid that cli module should not depend on cluster mode.

Reply:
If not, how can it use classes like ClusterMonitor?